shared/src/main/java/uk/gov/di/authentication/shared/entity/MFAMethod.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.di.authentication.shared.entity;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+
+import java.util.Map;
+import java.util.Objects;
+
+@DynamoDBDocument
+public class MFAMethod {
+
+    public static final String ATTRIBUTE_MFA_METHOD_TYPE = "MfaMethodType";
+    public static final String ATTRIBUTE_CREDENTIAL_VALUE = "CredentialValue";
+    public static final String ATTRIBUTE_ENABLED = "Enabled";
+    public static final String ATTRIBUTE_UPDATED = "Updated";
+
+    private String mfaMethodType;
```

Comment:
Any reason we can't set this directly as the `MFAMethodType` type?

Reply:
Going to look at this later on as there a couple of ways to do it, need to see what the best one is.  It will always be a String in the database anyway.